docs/stories/blocks/PromptBlock.stories.tsx

```
@@ -0,0 +1,67 @@
+      'div',
+      { className: 'flex gap-2 items-center' },
+      createElement('button', { 'aria-label': 'Send', type: 'button' }, '📤'),
+      createElement('button', { 'aria-label': 'Attach', type: 'button' }, '📎'),
+      createElement('button', { 'aria-label': 'Mic', type: 'button' }, '🎤')
+    ),
+  },
+};
+
+export const AutoResizing = {
+  render: args => {
+    id: 'prompt-block-autoResizing';
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)